src/components/ScreenStackItem.tsx

```diff
@@ -53,14 +56,14 @@ function ScreenStackItem({
       <DebugContainer
         style={[
           stackPresentation === 'formSheet'
-            ? Platform.OS === 'ios'
-              ? styles.absolute
-              : null
+            ? {
+                ...styles.sheet,
+                maxHeight: Platform.OS === 'android' ? '100%' : undefined,
```

Comment:
Does this break anything on iOS?

Reply:
I can't remember if there were any problems, can't find any right now. I believe it's just extra precaution.

Reply:
Having a single style would look neat though